README.md

```diff
@@ -320,6 +321,31 @@ end
 
 ```
 
+### Container Analysis (Alpha)
+
+- [google-cloud-container_analysis README](google-cloud-container_analysis/README.md)
+- [google-cloud-container_analysis API documentation](https://googleapis.github.io/google-cloud-ruby/docs/google-cloud-container_analysis/latest)
+- [google-cloud-container_analysis on RubyGems](https://rubygems.org/gems/google-cloud-container_analysis)
+- [Container Analysis documentation](https://cloud.google.com/container-registry/docs/container-analysis/)
+
+#### Quick Start
+
+```sh
+$ gem install google-cloud-container_analysis
+```
+
+#### Preview
+
+```ruby
+require "google/cloud/container_analysis"
+
+grafeas_client = Grafeas.new
+parent = Grafeas::V1::GrafeasClient.project_path my_project_id
```

Comment:
As a rule, code examples should be executable. In the handwritten packages we try to always use examples that are copied from API docs that are under doctest. Maybe  s/my_project_id/"my-project"/ ?

Reply:
Done.